text/000x-lowering-deposits-assethub.md

```diff
@@ -0,0 +1,197 @@
+# RFC-0000: Lowering NFT Deposits on Polkadot and Kusama Asset Hubs
+
+|                 |                                                                                                    |
+| --------------- | -------------------------------------------------------------------------------------------------- |
+| **Start Date**  | 2 November 2023                                                                                    |
+| **Description** | A proposal to reduce the minimum deposit required for collection creation on the Polkadot and Kusama Asset Hub, making it more accessible and affordable for artists. |
+| **Authors**     | [Aurora Poppyseed](https://github.com/poppyseedDev), [Just_Luuuu](https://github.com/justLuuuu), [Viki Val](https://github.com/vikiival)  |
+
+## Summary
+
+This RFC proposes changing the current deposit requirements on the Polkadot and Kusama Asset Hub for creating NFT collection, minting an individual NFT, and lowering it's coresponding metadata and attribute deposit. The objective is to lower the barrier to entry for NFT creators, fostering a more inclusive and vibrant ecosystem while maintaining network integrity and preventing spam.
+
+## Motivation
+
+The current deposit of 10 DOT for collection creation (along with 0.01 DOT for item deposit and 0.2 DOT for metadata and attribute deposit) on the Polkadot Asset Hub and 0.1 KSM on Kusama Asset Hub presents a significant financial barrier for many NFT creators. By lowering the deposit requirements, we aim to encourage more NFT creators to participate in the Polkadot NFT ecosystem, thereby enriching the diversity and vibrancy of the community and its offerings.
+
+The actual implementation of the deposit is an arbitrary number coming from [Uniques pallet](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs#L757). It is not a result of any economic analysis. This proposal aims to adjust the deposit from constant to dynamic pricing based on the `deposit` function with respect to stakeholders.
```

Comment:
This is a testnet runtime (and the line number on master is already out of date). Can you update to a release tag from the Polkadot runtime?

Reply:
```suggestion
The actual implementation of the deposit is an arbitrary number coming from [Uniques pallet]
(https://github.com/paritytech/polkadot-sdk/blob/afadb830e0a38365a5afec36fd1b2c77896a0a85/cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs#L756-L762). It is not a result of any economic analysis. This proposal aims to adjust the deposit from constant to dynamic pricing based on the `deposit` function with respect to stakeholders.
```

Reply:
Would that be branch `release-polkadot-v1.6.0` ?

Reply:
```suggestion
The actual implementation of the deposit is an arbitrary number coming from [Uniques pallet](https://github.com/polkadot-fellows/runtimes/blob/7515fc0385f8d48f2d08b2182a24174148e70135/system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs#L686-L692). It is not a result of any economic analysis. This proposal aims to adjust the deposit from constant to dynamic pricing based on the `deposit` function with respect to stakeholders.
```

Reply:
Or on [polkadot fellowship](https://github.com/polkadot-fellows/runtimes)?

Reply:
Yeah exactly, the Fellowship one